lib/llvmopencl/CanonicalizeBarriers.cc

```diff
@@ -147,8 +152,10 @@ CanonicalizeBarriers::ProcessFunction(Function &F) {
     // they just start several parallel regions. Simplifies
     // loop handling.
 
-    const bool HAS_NON_BRANCH_INSTRUCTIONS_AFTER_BARRIER = 
```

Comment:
Please check style and naming convention. We want to clean up the code base with new commits towards using LLVM style for C++ files and GNU C for C files. The patch formatter script might help.

Reply:
We clean up nearby/touched parts.